server/config/client.go

```diff
@@ -197,8 +199,10 @@ func GenerateClientConfig(c *model.Config, telemetryID string, license *model.Li
 		if *license.Features.DataRetention {
 			props["DataRetentionEnableMessageDeletion"] = strconv.FormatBool(*c.DataRetentionSettings.EnableMessageDeletion)
 			props["DataRetentionMessageRetentionDays"] = strconv.FormatInt(int64(*c.DataRetentionSettings.MessageRetentionDays), 10)
```

Comment:
Why does the client need the days? It it enough to send the days value?

Reply:
Yeah I guess I could just return the value for `GetMessageRetentionHours` to the client because it doesn't need to know if the value is coming from days or hours